projects/ngx-gauge/src/gauge/gauge.ts

```diff
@@ -185,7 +185,9 @@ export class NgxGauge implements AfterViewInit, OnChanges, OnDestroy {
     }
 
     private _clear() {
+      if(this._context) {
```

Comment:
I should put this line of code inside the `_drawShell()` method because it is using `_context' variable a lot. Also instead of checking on `_context`, I would like to check on `_initialized`, and if it still throws an error, then I would like to see how can I change `_initialized` bit to `true` only when `_context` is defined. Thanks for your pull request, it helped me a lot. Thanks!